orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -193,9 +193,9 @@ OpSchema& RegisterLambOpSchema(OpSchema&& op_schema) {
           {"tensor(float16)", "tensor(float)", "tensor(double)", "tensor(bfloat16)"},
           "Constrain input types to float tensors.")
       .TypeConstraint(
-          "T_FP16",
+          "T_MIXED_PRECISION_FP",
```

Comment:
Mixed precision training means "some ops are computed in float32, some other ops are computed in other precision, e.g. float16, bfloat16 for now, in the future, maybe int8". If we call it mixed precision data type, it seems confusing. Is it possible to find a better name?

Reply:
Any suggestion?

Reply:
no good suggestion.